### DISCUSSION TITLE
Add Publisher field

### DISCUSSION THREAD

**Discussion by exstntlmsnthrp** [mateusz-bak/openreads#715] [Ideas]
Love the app, I've been using it since you first put it on GitHub for public access. Thanks!

**Is your feature request related to a problem? Please describe.**
No, I would not specifically say this is a problem. I'd like to request that a "publisher" field be added. All books (including self published) have publishers listed and the publisher may vary depending on the year or country of publication, amongst other factors. This is a field in open library, but I don't know if the field isn't imported or it's just not visible in openreads. 

**Describe the solution you'd like**
I would like to see a publisher field as one of the default fields instead of having to add it as a custom field. I'd also like to see it auto populated when selecting a book from open library. 

**Describe alternatives you've considered**
The only other alternative is creating a custom field. However, this doesn't seem to get auto populated from open library. 

**Additional context**
If I find a book in my collection that has had multiple publishers, I'll add screenshots later.

**Comment by rafaelalvessa** [comment]
I've started working on this. I just need to add a few more things and figure out localisation before I create a pull request.

**Comment by rafaelalvessa** [comment]
Implemented in #858.
